Dockerfile

```diff
@@ -1,26 +1,24 @@
-FROM python:3-slim AS builder
-ENV MARK="9.9.0"
-
+FROM python:3.11-slim-bookworm AS builder
+ENV MARK="9.11.1"
 ADD . /app
 WORKDIR /app
-
 RUN pip install --target=/app -r requirements.txt && \
     apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y tar curl gnupg2 && \
     rm -rf /var/lib/apt/lists/* && \
     curl -LO https://github.com/kovetskiy/mark/releases/download/${MARK}/mark_Linux_x86_64.tar.gz && \
-    tar -xvzf mark_Linux_x86_64.tar.gz && chmod +x mark && mv mark /usr/local/bin/mark \
-    && curl -L https://dl-ssl.google.com/linux/linux_signing_key.pub |apt-key add - \
-    && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list \
-    && apt update && apt-get install -y google-chrome-stable
+    tar -xvzf mark_Linux_x86_64.tar.gz && chmod +x mark && mv mark /usr/local/bin/mark && rm mark_Linux_x86_64.tar.gz \
+    && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-FROM python:3-slim
+FROM chromedp/headless-shell:latest
```

Comment:
I'm not sure about this change instead of copy from builder...you are moving from 156MB of base image to more 285MB this will have a sensible impact on our build times
![image](https://github.com/draios/infra-action-mark2confluence/assets/19665312/35f0217a-7341-43a1-9da2-7c73fbf88820)

Reply:
Looks like this comment got marked as outdated.  I agree with mbovo on this.  No need to use a different/larger container.